x-pack/plugin/src/test/resources/rest-api-spec/test/index_lifecycle/30_rerun.yml

```diff
@@ -0,0 +1,141 @@
+---
+setup:
+  - do:
+      cluster.health:
+          wait_for_status: yellow
+
+  - do:
+      acknowlege: true
+      xpack.index_lifecycle.put_lifecycle:
+        lifecycle: "my_lifecycle"
+        body: |
+           {
+             "policy": {
+               "type": "timeseries",
+               "phases": {
+                 "warm": {
+                   "after": "1000s",
+                   "actions": {
+                     "forcemerge": {
+                       "max_num_segments": 10000
+                     }
+                   }
+                 },
+                 "hot": {
+                   "after": "1000s",
+                   "actions": { }
+                 }
+               }
+             }
+           }
+
+  - do:
+      acknowledge: true
+      xpack.index_lifecycle.get_lifecycle:
+        lifecycle: "my_lifecycle"
+
+---
+teardown:
+
+  - do:
+      acknowledge: true
+      indices.delete:
+        index: my_index
+
+  - do:
+      acknowledge: true
+      xpack.index_lifecycle.delete_lifecycle:
+        lifecycle: "my_lifecycle"
+
+  - do:
+      catch: missing
+      xpack.index_lifecycle.get_lifecycle:
+        lifecycle: "my_lifecycle"
+
+---
+"Test Basic Re-run":
```

Comment:
Hey @colings86, do you have an idea of which *real* policy configuration can deterministically lead us into an error-step? This test will never pass since the steps-registry will not return the error unless it *actually* happened.

Reply:
One way that I found accidentally is to configure a rollover action to rollover an alias that does not exist. I would expect that trying to shrink an index to a number of shards that it not a factor of the original number of shards would cause the error step too (e.g. try to shrink a 5 shard index to 2 shards).

Reply:
thanks, this scenario does work, but I cannot get the timing to work.

Reply:
I'll skip this for now